plotter/contour.go

```diff
@@ -122,6 +122,10 @@ const naive = false
 
 // Plot implements the Plot method of the plot.Plotter interface.
 func (h *Contour) Plot(c draw.Canvas, plt *plot.Plot) {
+	if h.Min > h.Max {
```

Comment:
perhaps we should document this in the `Controur` struct or the `NewContour` function ?

Reply:
Thanks. Done.